crates/nu-cli/src/commands/parse/command.rs

```diff
@@ -41,6 +41,19 @@ impl WholeStreamCommand for Command {
     ) -> Result<OutputStream, ShellError> {
         operate(args, registry).await
     }
+
+    fn examples(&self) -> Vec<Example> {
+        let mut row = IndexMap::new();
+        row.insert("foo".to_string(), Value::from("hi"));
+        row.insert("bar".to_string(), Value::from("there"));
+        let mut result_table = Vec::new();
+        result_table.push(UntaggedValue::row(row).into_untagged_value());
+        vec![Example {
+            description: "Parse a string into two named columns",
+            example: "echo \"hi there\" | parse \"{foo} {bar}\"",
+            result: Some(vec![UntaggedValue::Table(result_table).into()]),
```

Comment:
Parse will give you a row instead of a table here. Table (which should probably be called a "sub-table") is meant for when one table lives inside another. The top level data for Nu is a stream of row values.

Reply:
Thanks, only "returning" the row (instead of wrapping in another table) worked.